jenkins/scality-glance-store-functional-tests/20-install-openstack.sh

```diff
@@ -16,6 +16,13 @@ function common {
 
     if is_centos; then
         sudo yum install -y ftp://ftp.is.co.za/mirror/fedora.redhat.com/epel/6/x86_64/python-mox-0.5.3-2.el6.noarch.rpm
+
+        if test "${DEVSTACK_BRANCH:-x}" = "stable/juno"; then
+            RHEL6_RDO_REPO_RPM="http://buildlogs.centos.org/centos/6/cloud/x86_64/openstack-juno/centos-release-openstack-juno-2.el6.noarch.rpm"
+            export RHEL6_RDO_REPO_RPM
```

Comment:
Why not a one liner : export RHEL6_RDO_REPO_RPM="http://buildlogs.centos.org/centos/6/cloud/x86_64/openstack-juno/centos-release-openstack-juno-2.el6.noarch.rpm" ?